lib/launcher.js

```diff
@@ -54,6 +55,17 @@ class Launcher {
         await fs.writeFile(this.files.packageJSON, JSON.stringify(packageData, ' ', 2))
         await fs.rm(path.join(this.projectDir, '.config.nodes.json'), { force: true })
         await fs.rm(path.join(this.projectDir, '.config.nodes.json.backup'), { force: true })
+
+        // as the package.json file has been re-written, we will "assume" different versions
+        // of the modules and therefore remove the package-lock.json file to permit modules
```

Comment:
It may be worth noting that this is only because of the `--production` flag, a normal `npm install` would update the package-lock.